### PR TITLE
Backend/team size

### PR DESCRIPTION
**Pull request by gnoriega27**
## What issue is this PR solving?
Closes: #88 

## Description of change
Limit amount of players to desired amount, in our case the UI should pass either a 6 or 9 which is solved in issue #26 . Note that at the moment the clients cannot be spectators #74, the connection is refused. There is no UI at the moment.

## How to reproduce the changes
1. Set the number of players in the script "MainMenu.cs", Line 22,             ChangeRoomCapacity(<desired number 6 or 9>);

## Screenshots or GIFs
Screenshot (3)_LI
Example when setting max players to 2, you can see at the 'game' at the right the connection is refused.

### CODE REVIEW

**Comment by mjimenez98** [comment]
I am getting the following error which does not let me compile and run the game:
```
Assets/Scripts/Callbacks/TokenCallbacks.cs(10,37): error CS0246: 
The type or namespace name 'ConnectionToken' could not be found 
(are you missing a using directive or an assembly reference?)
```
I deleted the repo and recloned to see if it was something on my end, and I am still having the same issue. Can someone give it a try?

**Comment by gnoriega27** [comment]
@mjimenez98 Fixed

**Comment by stevau5** [comment]
Just ran it and all good on my end.. @gnoriega27 @mjimenez98

File: Assets/Scripts/UI/MainMenu.cs [hunk 1]
```diff
@@ -12,14 +12,19 @@ public class MainMenu : Bolt.GlobalEventListener
...
         private bool localTest;
 
+        [SerializeField]
+        private bool isPlayer = false;
+
         private void Awake()
         {
             if (localTest)
                 BoltLauncher.StartSinglePlayer();
+            ChangeRoomCapacity(2);
```
Comment (mjimenez98) [inline Assets/Scripts/UI/MainMenu.cs]: Should this value be changed to `roomPlayersCapacity` in ServerCallbacks?

**Comment by mjimenez98** [comment]
@stevau5 Thank you ❤️

File: Assets/Scripts/UI/MainMenu.cs (hunk 1 above)
Comment (yshrdbrn) [reply Assets/Scripts/UI/MainMenu.cs]: It's the other way around actually. When a player wants to be the server, he manually sets the capacity of his game (through `MainMenu.ChangeRoomCapacity()`) and it's automatically enforced after.

When the main menu supports it, the parameter for the function should come from the user. Till then though, I suggest we comment this line to not interfere with our testing.

Comment (gnoriega27) [reply Assets/Scripts/UI/MainMenu.cs]: Agreed. Will comment it out and then we should merge.

**Comment by gnoriega27** [comment]
@mjimenez98 I think this is already ready to be merged. Should I do it myself?

**Pull request merged**